src/main.c

```diff
@@ -149,6 +149,7 @@ int main(int argc, char **argv) {
     fprintf(stderr, "internal font error when starting the application\n");
   }
 
+  int has_restarted = 0;
```

Comment:
Is this ever updated anywhere? I can only see `RESTARTED` references in main.c and init.lua and `has_restarted` only in main.c

Reply:
Also good point. Fixed.